src/database/schema.rs

```diff
@@ -1,34 +1,34 @@
 table! {
     bundle (id) {
-        id -> Bpchar,
-        owner_address -> Nullable<Bpchar>,
-        block_height -> Int8,
+        id -> Nullable<Text>,
+        owner_address -> Nullable<Text>,
+        block_height -> Integer,
     }
 }
 
 table! {
     leaders (address) {
-        address -> Bpchar,
+        address -> Nullable<Text>,
     }
 }
 
 table! {
     transactions (id) {
-        id -> Bpchar,
-        epoch -> Int8,
-        block_promised -> Int8,
-        block_actual -> Nullable<Int8>,
-        signature -> Bytea,
-        validated -> Bool,
-        bundle_id -> Nullable<Bpchar>,
-        sent_to_leader -> Bool,
+        id -> Nullable<Text>,
+        epoch -> Integer,
+        block_promised -> Integer,
+        block_actual -> Nullable<Integer>,
+        signature -> Binary,
+        validated -> Integer,
+        bundle_id -> Nullable<Text>,
```

Comment:
Not exactly related to this PR, but how can we have transaction without bundle ID?

Reply:
Yes

It is bundled in intervals

Reply:
OK, thanks for explaining.